wagtail/core/models.py

```diff
@@ -1170,6 +1201,14 @@ def localized_draft(self):
 
     @property
     def localized(self):
+        """
+        Finds the translation in the current active language.
+
+        If there is no translation in the active language, self is returned.
+
+        Note: This will won't return the translation if it is in draft.
+        If you want to include drafts, uses the ``.localized_draft`` attribute instead.
```

Comment:
```suggestion
        Note: This will not return the translation if it is in draft.
        If you want to include drafts, use the ``.localized_draft`` attribute instead.
```